src/query/plan/variable_start_planner.cpp

```diff
@@ -209,19 +221,43 @@ CartesianProduct<VaryMatchingStart> VaryMultiMatchingStarts(const std::vector<Ma
   return MakeCartesianProduct(std::move(variants));
 }
 
+CartesianProduct<VaryMatchingStart> VaryFilterMatchingStarts(const Matching &matching,
+                                                             const SymbolTable &symbol_table) {
+  auto i = 0;
```

Comment:
I would change this to something more meaningful, since it reprsents all filter matchings counter, then to:
`all_filter_matchings_cnt`

Reply:
makes sense